src/EditorFeatures/Core/ExternalAccess/VSTypeScript/VSTypeScriptInProcLanguageClient.cs

```diff
@@ -56,21 +56,11 @@ public override ServerCapabilities GetCapabilities(ClientCapabilities clientCapa
 
         serverCapabilities.ProjectContextProvider = true;
 
-        var isPullDiagnostics = GlobalOptions.IsLspPullDiagnostics();
```

Comment:
similarly, IsLspPullDiagnostics would effectively now always return 'true'.  so this was removed, and the impact inlined.